configs/obsolete/ttalps_plotter_signal_config.py

```diff
@@ -181,6 +181,8 @@
   LLPnanoAOD_histograms += (
     Histogram(muonVertexCollectionName+"_Lxy"                  , "", False, True  , default_norm        , 20 , 0     , 800   , 1e-4  , 1e9   , "#mu vertex L_{xy} [cm]"                 , "# events (2018)"   ),
     Histogram(muonVertexCollectionName+"_logLxy"               , "", False, True  , default_norm        , 10 , -5    , 3     , 1e-4  , 1e9   , "#mu vertex log_{10}(L_{xy}) [cm]"                 , "# events (2018)"   ),
+    Histogram(muonVertexCollectionName+"_invMass"              , "", False, True  , default_norm        , 10, 0      , 10    , 1e-5  , 1e6   , "#mu vertex M_{#mu #mu} [GeV]"           , "# events (2018)"   ),
```

Comment:
The idea would be to remove these obsolete plotter configs - do you need them for anything?